public/index.html

```diff
@@ -1,18 +1,90 @@
+                    <div v-if="!currentClimber" class="header gray">NO INFORMATION FOUND</div>
+                    <div v-if="!currentClimber" class="no-info">
+                        <p class="">There is no climbing card information linked to the ID number</p>
+                        <img src="assets/not-found-illustration.svg" alt="">
+                    </div>
+
+                    <div v-if="currentClimber" id="result-content">
+                        <div class="row">
+                            <p class="heading">ISIKUKOOD</p>
+                            <p class="content">{{currentClimber.id}}</p>
+                        </div>
+                        <div class="row">
+                            <p class="heading">TÄISNIMI</p>
+                            <p class="content">{{currentClimber.name}}</p>
+                        </div>
+                        <div class="row">
+                            <p class="heading">EKSAMI AEG</p>
+                            <p class="content">2020/23/02</p>
+                        </div>
+                        <div class="row">
+                            <p class="heading">EKSAMINEERIJA</p>
+                            <p class="content">PEEDO</p>
+                        </div>
+                        <p class="description">Ronimisliit kinnitab, et ronijakaardi omanikul on piisavad osused iseseisvaks ronimiseks ja julgestamiseks kaljuronimisseinal ülaltjulgestuses.</p>
+                    </div>
+                </div>
+                <p v-if="!showInstructions" class="warning">Validate the card information with the ID data of the climber</p>
+                <p v-if="!showInstructions" class="warning">Make sure the climber and the card holder is the same person</p>
+                <div v-if="showInstructions" class="instructions">
+                    <div class="row">
+                        <div class="row-number-wrapper">
+                            <div class="row-number">1</div>
+                        </div>
+                        <p>Request ID to climbers who want to access the climbing wall.</p>
+                    </div>
+                    <div class="row">
+                        <div class="row-number-wrapper">
+                            <div class="row-number">2</div>
+                        </div>
+                        <p>Search the climbers certification with the ID Number.</p>
+                    </div>
+                    <div class="row">
+                        <div class="row-number-wrapper">
+                            <div class="row-number">3</div>
+                        </div>
+                        <p>Validate result the ID data and person’s picture.</p>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+    </div>
 </body>
-</html>
+
+<script type="module" src="./index.js"></script>
+
+</html>
```

Comment:
Add line breaks to the last lines for every file.

Reply:
Should be fixed